code/script.js

```diff
@@ -244,100 +277,171 @@ const generateBoard = () => {
   })
 }
 
-// Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
   secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
 }
 
-// This function to start (and restart) the game
 const start = () => {
-  // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
+  playerName= prompt('Hi! What´s your name?')
+  setSecret()
+  generateBoard()
+  attributeQuestions = 0
+  winOrLose.classList.remove('active')
+  board.classList.remove('hide')
+  winOrLoseText.innerHTML = `
+    `   
 }
 
-// setting the currentQuestion object when you select something in the dropdown
 const selectQuestion = () => {
   const category = questions.options[questions.selectedIndex].parentNode.label
-  // This variable stores what option group (category) the question belongs to.
-  // We also need a variable that stores the actual value of the question we've selected.
-
+  const userInput = questions.options[questions.selectedIndex].value
+  
   if (category === 'hair color') {
     currentQuestion = {
       attribute: 'hairColor',
-      // value: ,
-      // 👆 add the value from the input here
-      category: category,
+      value: userInput,      
+      category: category,      
```

Comment:
remember that in situations like `category: category, ` you can just use `category,` 🤓